frontend/src/components/regions/AgileBoard/AgileCard/AgileCardActions/utils.js

```diff
@@ -35,13 +36,16 @@ export function getTeamPermissions({ authUser, viewedUser }) {
 
 function getShowAddReviewButton({ card, permissions, isReviewer }) {
   const REVIEW_STATUSES = [IN_REVIEW, COMPLETE, REVIEW_FEEDBACK];
-  if (isReviewer && REVIEW_STATUSES.indexOf(card.status) !== -1) return true;
+  if(isReviewer && REVIEW_STATUSES.indexOf(card.status) !== -1){
+    if(permissions[VIEW_ONLY]) return false;
```

Comment:
This logis isn't correct. Sometimes a person can be given multiple permissions. They dont cancel each other out, they stack up.

If you have permission to use the blue pen and I give you permission to use the red pen as well, you can still use the blue pen. 

`getShowAddReviewButtons` looks like it works fine. 

Take a look at `src/components/regions/CardDetails/index.js`.   

There's a piece of code in there that is wrong:
```
  const showAddReviewButton =
    (isReviewer || isStaff) &&
    [IN_REVIEW, COMPLETE, REVIEW_FEEDBACK].indexOf(projectCardStatus) !== -1;
```
Can you fix this code.